### PR TITLE
enable onefile EXE compilation with PyInstaller

### PR DESCRIPTION
**Pull request by kenvenner**
here are the small changes talked about in the issue - and now I learned how to fork and PR - thanks

### CODE REVIEW

File: zipcodes/__init__.py [hunk 1]
```diff
@@ -24,11 +24,21 @@
 __email__ = "pianka@eml.cc"
 __license__ = "MIT"
 __package__ = "zipcodes"
-__version__ = "1.1.2"
+__version__ = "1.1.2a"
```
Comment (seanpianka) [inline zipcodes/__init__.py:27]: Can you change this to 1.1.3? I think this warrants a patch version bump.

Comment (kenvenner) [reply zipcodes/__init__.py]: This requested change has been made

Comment (kenvenner) [reply zipcodes/__init__.py]: Sean - just want to validate that you can see the version change - I committed the change - but not sure if I did it correctly - so just anted to validate you can see the 2nd change I made.

Comment (seanpianka) [reply zipcodes/__init__.py]: Yes, I see it in the Python file. There is also a file which contains the version separately, could you update this to 1.1.3 as well? That's all left before I can publish a new version.

Thanks for your PR.

**Comment by kenvenner** [comment]
wondering if you were going to merge this pull request?

**Pull request merged**